<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticeDlg</class>
 <widget class="QDialog" name="NoticeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
border-top-left-radius:5px;
border-top-right-radius:5px;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>5</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;
color:rgb(255,255,255);
border:none</string>
      </property>
      <property name="text">
       <string>提示</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_btnClose">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>0</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(255,0,0);
border-top-left-radius:0px;
border-top-right-radius:5px;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-image: url(:/images/弹窗X选-02.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="m_labelInfo1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>该资源库下尚存在人脸照片</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="m_btnOk">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>102</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;
border-radius:3px;
border:1px solid rgb(180,180,180)</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_labelInfo2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>65</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font: 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>请先清除照片后再执行删除操作！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
